<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaceClass</class>
 <widget class="QMainWindow" name="interfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="AbulEduStackedWidgetV1" name="stackedWidget">
      <widget class="QWidget" name="mainPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="fete">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="optimizationFlags">
           <set>QGraphicsView::DontAdjustForAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="AbulEduAproposV1" name="aboutPage"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnInitialise">
      <property name="text">
       <string>Initialiser &amp;Paramètres</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionAfficher_l_diteur"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuExercices">
    <property name="title">
     <string>Exercices</string>
    </property>
    <widget class="QMenu" name="menuTables_d_addition">
     <property name="title">
      <string>Tables d'addition</string>
     </property>
     <addaction name="actionp2"/>
     <addaction name="actionp3"/>
     <addaction name="actionp4"/>
     <addaction name="actionp5"/>
     <addaction name="actionp6"/>
     <addaction name="actionp7"/>
     <addaction name="actionp8"/>
     <addaction name="actionp9"/>
    </widget>
    <widget class="QMenu" name="menuTables_de_multiplication">
     <property name="title">
      <string>Tables de multiplication</string>
     </property>
     <addaction name="actionX_2"/>
     <addaction name="actionX_3"/>
     <addaction name="actionX_4"/>
     <addaction name="actionX_5"/>
     <addaction name="actionX_6"/>
     <addaction name="actionX_7"/>
     <addaction name="actionX_8"/>
     <addaction name="actionX_9"/>
    </widget>
    <widget class="QMenu" name="menuMultiples">
     <property name="title">
      <string>Multiples</string>
     </property>
     <addaction name="actionDe_5"/>
     <addaction name="actionDe_10"/>
     <addaction name="actionDe_15"/>
     <addaction name="actionDe_20"/>
     <addaction name="actionDe_25"/>
     <addaction name="actionDe_50"/>
    </widget>
    <widget class="QMenu" name="menuCompl_ments">
     <property name="title">
      <string>Compléments</string>
     </property>
     <addaction name="action_10"/>
     <addaction name="action_100"/>
     <addaction name="action_20"/>
     <addaction name="actionMaison_des_nombres"/>
    </widget>
    <widget class="QMenu" name="menuOrdres_de_grandeur">
     <property name="title">
      <string>Ordres de grandeur</string>
     </property>
     <addaction name="actionSur_des_additions"/>
     <addaction name="actionSur_des_soustractions"/>
     <addaction name="actionSur_des_multiplications"/>
    </widget>
    <addaction name="actionAdditions"/>
    <addaction name="actionMultiplications"/>
    <addaction name="actionSoustractions"/>
    <addaction name="menuTables_d_addition"/>
    <addaction name="menuTables_de_multiplication"/>
    <addaction name="menuMultiples"/>
    <addaction name="menuCompl_ments"/>
    <addaction name="menuOrdres_de_grandeur"/>
   </widget>
   <widget class="QMenu" name="menuLangues">
    <property name="title">
     <string>Langues</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionVerrouillage_nombres"/>
   </widget>
   <widget class="QMenu" name="menu_Utilisateur">
    <property name="title">
     <string>&amp;Utilisateur</string>
    </property>
    <addaction name="action_Journal_de_mes_activit_s"/>
    <addaction name="action_Changer_d_utilisateur"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuExercices"/>
   <addaction name="menuLangues"/>
   <addaction name="menuOptions"/>
   <addaction name="menu_Utilisateur"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdditions">
   <property name="text">
    <string>Additions</string>
   </property>
  </action>
  <action name="actionMultiplications">
   <property name="text">
    <string>Multiplications</string>
   </property>
  </action>
  <action name="actionp2">
   <property name="text">
    <string notr="true">+ 2</string>
   </property>
  </action>
  <action name="actionp3">
   <property name="text">
    <string notr="true">+ 3</string>
   </property>
  </action>
  <action name="actionp4">
   <property name="text">
    <string notr="true">+ 4</string>
   </property>
  </action>
  <action name="actionp5">
   <property name="text">
    <string notr="true">+ 5</string>
   </property>
  </action>
  <action name="actionp6">
   <property name="text">
    <string notr="true">+ 6</string>
   </property>
  </action>
  <action name="actionp7">
   <property name="text">
    <string notr="true">+ 7</string>
   </property>
  </action>
  <action name="actionp8">
   <property name="text">
    <string notr="true">+ 8</string>
   </property>
  </action>
  <action name="actionp9">
   <property name="text">
    <string notr="true">+ 9</string>
   </property>
  </action>
  <action name="actionX_2">
   <property name="text">
    <string notr="true">x 2</string>
   </property>
  </action>
  <action name="actionX_3">
   <property name="text">
    <string notr="true">x 3</string>
   </property>
  </action>
  <action name="actionX_4">
   <property name="text">
    <string notr="true">x 4</string>
   </property>
  </action>
  <action name="actionX_5">
   <property name="text">
    <string notr="true">x 5</string>
   </property>
  </action>
  <action name="actionX_6">
   <property name="text">
    <string notr="true">x 6</string>
   </property>
  </action>
  <action name="actionX_7">
   <property name="text">
    <string notr="true">x 7</string>
   </property>
  </action>
  <action name="actionX_8">
   <property name="text">
    <string notr="true">x 8</string>
   </property>
  </action>
  <action name="actionX_9">
   <property name="text">
    <string notr="true">x 9</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A &amp;propos</string>
   </property>
  </action>
  <action name="actionDocumentation_utilisateurs">
   <property name="text">
    <string>Documentation utilisateurs</string>
   </property>
  </action>
  <action name="actionSoustractions">
   <property name="text">
    <string>Soustractions</string>
   </property>
  </action>
  <action name="actionDe_5">
   <property name="text">
    <string>de 5</string>
   </property>
  </action>
  <action name="actionDe_10">
   <property name="text">
    <string>de 10</string>
   </property>
  </action>
  <action name="actionDe_15">
   <property name="text">
    <string>de 15</string>
   </property>
  </action>
  <action name="actionDe_20">
   <property name="text">
    <string>de 20</string>
   </property>
  </action>
  <action name="actionDe_25">
   <property name="text">
    <string>de 25</string>
   </property>
  </action>
  <action name="actionDe_50">
   <property name="text">
    <string>de 50</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>à 10</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>à 1000</string>
   </property>
  </action>
  <action name="action_100">
   <property name="text">
    <string>à 100</string>
   </property>
  </action>
  <action name="action_un_nombre_al_atoire">
   <property name="text">
    <string>à un nombre aléatoire</string>
   </property>
  </action>
  <action name="actionSur_des_additions">
   <property name="text">
    <string>sur des additions</string>
   </property>
  </action>
  <action name="actionSur_des_soustractions">
   <property name="text">
    <string>sur des soustractions</string>
   </property>
  </action>
  <action name="actionSur_des_multiplications">
   <property name="text">
    <string>sur des multiplications</string>
   </property>
  </action>
  <action name="actionMaison_des_nombres">
   <property name="text">
    <string>Maison des nombres</string>
   </property>
  </action>
  <action name="actionVerrouillage_nombres">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verrouillage nombres</string>
   </property>
  </action>
  <action name="action_Journal_de_mes_activit_s">
   <property name="text">
    <string>&amp;Journal de mes activités</string>
   </property>
  </action>
  <action name="action_Changer_d_utilisateur">
   <property name="text">
    <string>&amp;Changer d'utilisateur</string>
   </property>
  </action>
  <action name="actionAfficher_l_diteur">
   <property name="text">
    <string>Afficher l'éditeur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AbulEduAproposV1</class>
   <extends>QWidget</extends>
   <header>abuleduaproposv1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AbulEduStackedWidgetV1</class>
   <extends>QStackedWidget</extends>
   <header>abuledustackedwidgetv1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
